ui-components/src/locales/general.json

```diff
@@ -692,6 +692,10 @@
     "noData": "No data available."
   },
   "footer": {
+    "srHeading": "Footer",
+    "srProjectInformation": "Project information",
+    "srContactInformation": "Contact information",
+    "srLegalInformation": "Legal information",
```

Comment:
What does "sr" mean in this context? I'm wondering if it might be better to just put a longer name here so it's more obvious what it is to future maintainers.

Reply:
"sr" means screen reader. The abbreviation is used by frameworks like [tailwind ](https://tailwindcss.com/docs/screen-readers)and [bootstrap](https://getbootstrap.com/docs/4.0/utilities/screenreaders/). The tailwind class "sr-only" is used in this codebase as well. Let me know if you think I should still write out the longer name.

Reply:
Ok, that's fine - probably just my unfamiliarity.